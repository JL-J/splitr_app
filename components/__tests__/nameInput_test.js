import React from 'react';
import { TextInput } from 'react-native';
import { shallow } from 'enzyme';
import renderer from 'react-test-renderer';
import Input from '../input';

describe('<NameInput />', function() {
  it('renders the component correctly', () => {
    const snap = renderer.create(
      <Input />).toJSON();
    expect(snap).toMatchSnapshot();
  });

  it('renders one text input component', () => {
    const wrapper = shallow(<Input />);
    expect(wrapper.find(TextInput)).toHaveLength(1);
  });

});
