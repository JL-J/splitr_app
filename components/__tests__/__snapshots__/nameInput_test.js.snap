// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<NameInput /> renders the component correctly 1`] = `
<View
  style={
    Object {
      "padding": 12,
    }
  }
>
  <TextInput
    allowFontScaling={true}
    style={
      Object {
        "borderColor": "gray",
        "borderWidth": 1,
        "height": 40,
      }
    }
    underlineColorAndroid="transparent"
  />
</View>
`;
